src/main/java/org/broadinstitute/dsm/route/DashboardRoute.java

```diff
@@ -815,4 +792,17 @@ public void getKitRequestInformationPerMonth(@NonNull String realm, @NonNull Kit
             throw new RuntimeException("Error getting kit request information for download ", results.resultException);
         }
     }
+
+    private int getKitTypeId (@NonNull String realm, @NonNull KitType kitType) {
+        int kitTypeId = kitType.getKitId();
+        List<KitSubKits> subKits = KitSubKits.getSubKits(realm, kitType.getName());
+        if (subKits != null && !subKits.isEmpty()) {
+            //kit has sub kits (assumption: all subkits stay together and will therefore be counted as just "one" kit)
+            KitSubKits firstSubKit = subKits.get(0);
```

Comment:
so it always returns the first subkit?

Reply:
it only uses the kittypeId from the first subkit to count the numbers of subkits in the database